datafusion/core/tests/parquet/arrow_statistics.rs

```diff
@@ -1168,7 +1165,7 @@ async fn test_struct() {
         reader: reader.build().await,
         expected_min: Arc::new(struct_array(vec![(Some(1), Some(6.0), Some(12.0))])),
         expected_max: Arc::new(struct_array(vec![(Some(2), Some(8.5), Some(14.0))])),
-        expected_null_counts: UInt64Array::from(vec![0]),
+        expected_null_counts: UInt64Array::from(vec![2]),
```

Comment:
Something isn't right here -- the structs themselves aren't nullable (only some of their child fields)

https://github.com/apache/datafusion/blob/7c08a6fd00fcb352c11889ed62de7c9948978c79/datafusion/core/tests/parquet/mod.rs#L954-L958

https://github.com/apache/datafusion/blob/7c08a6fd00fcb352c11889ed62de7c9948978c79/datafusion/core/tests/parquet/mod.rs#L1024-L1045